decidim-core/db/migrate/20170221094835_add_scopes_to_processes.rb

```diff
@@ -0,0 +1,5 @@
+class AddScopesToProcesses < ActiveRecord::Migration[5.0]
+  def change
+    add_column :decidim_participatory_processes, :scope_ids, :integer, array: true, default: []
```

Comment:
Because I hate HABTM, and we won't need to find the relation from the other end (eg `scope.participatory_processes`).